### ISSUE TITLE
TypeError: Cannot read property 'lastModified' of null

### ISSUE THREAD

**Issue by OliverwengFiltered** [jdesboeufs/connect-mongo#392]
bug report

I followed the  migration doc, however, I'm getting the above error on logout.

here is my code to initialize the session with mongostore.

![Screen Shot 2021-02-23 at 9 37 52 AM](https://user-images.githubusercontent.com/28814939/108859061-d736ed00-75ba-11eb-872e-98545a6f18a8.png)

and here is the code for log out.

and I'm doing 
` req.session.destroy()` and `req.logout()` on logout action.

and getting this error on logout.
```
TypeError: Cannot read property 'lastModified' of null
09:32:42 0|web                | 2021-02-23T09:32:42:     at /Users/oliverweng/Desktop/DevelopmentApps/Filtered/node_modules/connect-mongo/build/main/lib/MongoStore.js:174:60
```

**Comment by mingchuno** [comment]
@OliverwengFiltered Ya, `lastModified` part got some bug. It is also reported in #390 and I plan to fix it in v4.2.0.
Can you please try to add the env `DEBUG=connect-mongo` to have a more detailed log?

**Comment by mingchuno** [comment]
This commit should fix the issue https://github.com/jdesboeufs/connect-mongo/commit/46a40db8a6d967ba0b1c2ad03b7281f832267f20

**Comment by mingchuno** [comment]
released in 4.2.0
